blaze/compute/pandas.py

```diff
@@ -165,6 +165,21 @@ def compute_up(expr, data, **kwargs):
     return compute_up(expr._child.distinct().count(), data, **kwargs)
 
 
+@dispatch(Repeat, Series)
+def compute_up(expr, data, **kwargs):
+    return expr.lhs.str.repeat(data)
+
+
+@dispatch(Concat, Series)
+def compute_up(expr, data, **kwargs):
+    return expr.lhs.str.cat(data)
+
+
+@dispatch(Interp, Series)
```

Comment:
i'd probably implement this like this:

```python
return compute(expr, data.values)
```
because pandas doesn't implement `%` symmetrically.